examples/caddy2/Caddyfile

```diff
@@ -185,9 +212,9 @@ element.DOMAIN.tld {
 #
 #        # Prevent some browsers from MIME-sniffing a response away from the declared Content-Type
 #        X-Content-Type-Options "nosniff"
-#
-#        # Disallow the site to be rendered within a frame (clickjacking protection)
-#        X-Frame-Options "SAMEORIGIN"
+
+#        # Only allow same base domain to render this website in a frame; Can be removed if the client (Element for example) is hosted on another domain
+#        # Content-Security-Policy frame-ancestors https://*.DOMAIN.tld
```

Comment:
The comment says "Only allow the base domain", but.. `*.DOMAIN.tld` seems to say "all sibling domains".

This seems to change the behavior compared to before - and to a more insecure way of doing things.

Previously, by default, **no framing was allowed at all**. Now, this `Content-Security-Policy` is there, but it's commented out by default, so **framing is allowed from anywhere**.

Reply:
Maybe "base domain" could be rephrased. The goal here is to allow the Jitsi integration in Element to work.

It is less secure, but should still be secure enough since you usually own the whole domain. It could be changed to specifically allow `https://element.DOMAIN.tld` only for a more secure default/example or subdomain renting services.

Reply:
To clarify, `X-Frame-Options "SAMEORIGIN"` specifically only allows `https://jitsi.DOMAIN.tld` to frame `jitsi.DOMAIN.tld`,

Reply:
It's still commented out by default though and I suspect most peope copy-pasting the file may leave it as it is (commented out), thus being exposed to framing from anywhere.

It's not the end of the world, but.. it is very different compared to what we had as a default before (no framing allowed from anywhere).

Reply:
That's a mistake on my end. I will fix it for dimension